frontend/src/Components/Label.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import React, { ComponentProps } from 'react';
+import { kinds, sizes } from 'Helpers/Props';
+import styles from './Label.css';
+
+export type LabelProps = {
```

Comment:
Exporting types makes component overrides easier

Reply:
An interface works fine as well

```
export interface LabelProps extends ComponentProps<'span'> {
  kind?: Extract<(typeof kinds.all)[number], keyof typeof styles>;
  size?: Extract<(typeof sizes.all)[number], keyof typeof styles>;
  outline?: boolean;
}
```

Though reducing it to only props that we actually expect makes the interface clearer, which I believe is limited to `title`.

Reply:
Be sure to keep in mind that since the interface is exported, any other interface with the same name will be merged

From experience, keeping props to the bare minimum leads to countless other unnecessary PRs do add other props one by one (title, name, id, autofocus.... etc)
Keeping the props open makes the component more flexible and allows it to be independent of the project it is used in